playground/infrastructure/helper.py

```diff
@@ -331,6 +333,17 @@ def _validate(tag: dict, supported_categories: List[str]) -> bool:
             category,
             category)
         valid = False
+
+  # check that context line's value is integer
+  context_line = tag.get(TagFields.context_line)
+  if not isinstance(context_line, int):
+      logging.error(
+          "tag's field context_line is incorrect: %s \n"
+          "context_line variable should be integer format, "
+          "but tag contains: %s",
```

Comment:
do we have to make this multi-lines error?

Reply:
@daria-malkova multi-lines error because the errors written above are also in this style.